src/main/java/Duke.java

```diff
@@ -1,10 +1,142 @@
+import java.util.Scanner;
+
+
 public class Duke {
+
+    private static Task[] tasks = new Task[100];
+    private static int taskCounter =0;
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
+
+        startMsg();
+        Scanner input = new Scanner(System.in);
+
+        while (input.hasNextLine()) {
+
+            String command = input.nextLine().trim();
+            String[] doneTokens = command.split(" ");
+            if (command.equals("")) continue;
+
+            if(command.equals("bye")) {
+                byeMsg();
+                break;
+            }
+            else if(command.equals("list")){
+                printList();
+            }
+            else if (doneTokens[0].equals("done")){
+                checkValidIndex(doneTokens[1]);
+            }
+            else {
+                //this switch statement checks the nature of the task
+                switch (doneTokens[0]){
+                case ("todo"):
+                    //obtains description of todo task
+                    String dsc = command.substring(command.indexOf(" ") + 1);
```

Comment:
Unreadable variable name detected. What is dsc? The string name should be more readable.